libs/sync/sync.go

```diff
@@ -34,6 +37,8 @@ type SyncOptions struct {
 	CurrentUser *iam.User
 
 	Host string
+
+	OutputHandler OutputHandler
```

Comment:
I see how this is necessary in the current setup (where we have no good means to pass this information from both the sync command and the deploy command). In some future we'll want to decouple this again and deal with output from the caller, passing only the channel with events.

Reply:
Output handler accepts a channel with events. It's set by the caller, so the caller is in control on how to deal with events. We can turn things around and create events channel on the caller side, passing it to the sync logic, not sure if there are any practical benefits of that